src/txn/txn.c

```diff
@@ -803,13 +803,13 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
 	int64_t resolved_update_count, visited_update_count;
 	uint32_t fileid;
 	u_int i;
-	bool locked, prepare, readonly, update_timestamp;
+	bool locked, prepare, readonly, update_timestamp, skip_update_assert;
```

Comment:
Small nit pick: Ideally, variable names should be in the alphabetic order. Sorry I did not notice before.